package.json

```diff
@@ -2,14 +2,14 @@
   "name": "netlify-function-tests",
   "version": "0.2.0",
   "scripts": {
-    "dev": "netlify dev --context production",
+    "dev": "vite dev",
     "build": "vite build"
   },
-  "devDependencies": {
+  "dependencies": {
```

Comment:
"dependencies" are bundled to functions ?

Reply:
This was just a "precautionary" measure. Both should work fine. Netlify sets `NODE_ENV` as `development` by default (during the builds). Due to this, all dependencies listed in `devDependencies` are also installed. Any installed (and required) module should be bundled in the Functions, so this change should not matter, but I didn't want to take chances.